var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -15,7 +15,6 @@
 import os
 import platform
 import re
-import sys
```

Comment:
Why was this removed? `sys` is referenced below this:

`is_windows = sys.platform == 'win32'`